scripts/jenkins-yoctobuild-build.sh

```diff
@@ -271,22 +271,6 @@ cd $WORKSPACE
 # Build server repositories.
 # ---------------------------------------------------
 
-# Build Go repositories.
-export GOPATH="$WORKSPACE/go"
-(
-    cd $WORKSPACE/go/src/github.com/mendersoftware/mender-artifact
-    make install
-    if grep -q build-natives-contained Makefile; then
-        make build-natives-contained
-    fi
```

Comment:
Won't this break when you test old Yocto branches using Jenkins?

Reply:
Jenkins uses [mender-qa/jenkins](https://github.com/mendersoftware/mender-qa/blob/jenkins/scripts/jenkins-yoctobuild-build.sh) branch, so that should be okey. We should be encouraged to do clean-ups here when possible.